backend/lcfs/web/api/email/services.py

```diff
@@ -0,0 +1,155 @@
+import os
+from lcfs.db.models.notification.NotificationType import NotificationTypeEnum
+import requests
+import structlog
+from fastapi import Depends
+from jinja2 import Environment, FileSystemLoader
+from typing import Dict, List, Any
+from datetime import datetime
+
+from lcfs.web.api.email.repo import CHESEmailRepository
+from lcfs.web.core.decorators import service_handler
+from lcfs.web.api.email.template_mapping import TEMPLATE_MAPPING
+
+logger = structlog.get_logger(__name__)
+
+
+class CHESEmailService:
+    """
+    Service layer for sending email notifications via CHES.
+    Handles CHES-specific logic and coordinates with the repository.
+    """
+
+    def __init__(self, repo: CHESEmailRepository = Depends()):
+        self.repo = repo
+
+        # CHES configuration
+        self.config = {
+            "AUTH_URL": os.getenv("CHES_AUTH_URL", ""),
```

Comment:
Currently all of our ENV vars start with LCFS_ and are mapped into settings.py, should we follow that pattern here?

Reply:
Ah nvm, I see how this works. We have a generic default template for some notifications but others have specific content.

Reply:
Thanks for your review, I've made the necessary changes as per your suggestions.